modern/src/runtime/AnimatedLayer.ts

```diff
@@ -63,23 +158,19 @@ class AnimatedLayer extends Layer {
   }
 
   setstartframe(framenum: number) {
-    unimplementedWarning("setstartframe");
-    return;
+    this.attributes.start = framenum;
   }
 
   setendframe(framenum: number) {
-    unimplementedWarning("setendframe");
-    return;
+    this.attributes.end = framenum;
   }
 
   setautoreplay(onoff: boolean) {
-    unimplementedWarning("setautoreplay");
-    return;
+    this.attributes.autoreplay = onoff;
```

Comment:
If you turn this on and the animation is stopped, does it start the animation?

Reply:
I'll need to test it in winamp, but I'm assuming not